Sources/Sentry/SentryDescriptor.m

```diff
@@ -0,0 +1,86 @@
+#import "SentryDescriptor.h"
+#import "SentryDefines.h"
+#import "SentryDemangler.h"
+
+static SentryDescriptor *_globalDescriptor;
+
+NSString *
+sentry_getClassDescription(Class aClass)
+{
+    return [_globalDescriptor getClassDescription:aClass];
+}
+
+NSString *
+sentry_getObjectClassDescription(NSObject *object)
+{
+    return [_globalDescriptor getObjectClassDescription:object];
+}
+
+NSString *
+sentry_getDescription(NSObject *object)
+{
+    return [_globalDescriptor getDescription:object];
+}
+
+void
+sentry_setGlobalDescriptor(SentryDescriptor *descriptor)
+{
+    _globalDescriptor = descriptor ?: [[SentryDescriptor alloc] init];
+}
+
+@implementation SentryDescriptor {
+#if SENTRY_HAS_UIKIT && TARGET_OS_MACCATALYST == 0
+    SentryDemangler *demangler;
+#endif
+}
+
++ (void)load
+{
+    _globalDescriptor = [[SentryDescriptor alloc] init];
+    NSLog(@"%d", TARGET_OS_MACCATALYST);
+}
+#if SENTRY_HAS_UIKIT && TARGET_OS_MACCATALYST == 0
```

Comment:
`m`: I think we should add a comment on why we only use the demangler for iOS, tvOS, and Catalyst.